### ISSUE TITLE
Can I run  a copy task after another copy task?

### ISSUE THREAD

**Issue by Jocs** [webpack-contrib/copy-webpack-plugin#301]
By looking at the source code, it is found that the tasks in the current patterns are all running in parallel, but I have a special requirement. I want to run another task after one task is finished. Can this be done with this plugin?

```javascript
new CopyWebpackPlugin([
      {
        from: path.resolve(__dirname, '../src/muya/themes'), // task 1
        to: path.join(__dirname, '../static/themes'),
        force: true
      },
      {
        from: path.join(__dirname, '../static'), // task 2
        to: path.join(__dirname, '../dist/electron/static'),
        ignore: ['.*']
      }
    ]),
```
I want to task 2 run after task 1.

**Comment by evilebottnawi** [comment]
@Jocs No, it is very strange if you want to copy one task after another, can you show real use case?

**Comment by Ohar** [comment]
@evilebottnawi 
1. Build app into `/dist`.
2. Copy static files into `/dist.
3. Copy `/dist` folder into `/docs` to update GitHub Pages.

**Comment by evilebottnawi** [comment]
@Ohar it is feature, feel free to send a PR, we need hook on `done` event

**Comment by evilebottnawi** [comment]
Close in favor https://github.com/webpack-contrib/copy-webpack-plugin/issues/318 (see https://github.com/webpack-contrib/copy-webpack-plugin/issues/318#issuecomment-458909457)
